schema/type_descriptions.yaml

```diff
@@ -54,30 +79,39 @@
 - type: DetectorAttributes
   property_descriptions:
     included: >
-      Configure list of attribute key patterns to include from resource detectors. If not set, all attributes are included.
+      Configure list of attribute key patterns to include from resource detectors.
       
       Attribute keys from resource detectors are evaluated to match as follows:
        * If the value of the attribute key exactly matches.
        * If the value of the attribute key matches the wildcard pattern, where '?' matches any single character and '*' matches any number of characters including none.
+      If omitted, all attributes are included.
     excluded: >
       Configure list of attribute key patterns to exclude from resource detectors. Applies after .resource.detectors.attributes.included (i.e. excluded has higher priority than included).
       
       Attribute keys from resource detectors are evaluated to match as follows:
        * If the value of the attribute key exactly matches.
        * If the value of the attribute key matches the wildcard pattern, where '?' matches any single character and '*' matches any number of characters including none.
+      If omitted, .included attributes are included.
   path_patterns:
     - .resource.detectors.attributes
 
 - type: AttributeLimits
   property_descriptions:
-    attribute_value_length_limit: Configure max attribute value size.
-    attribute_count_limit:  Configure max attribute count.
+    attribute_value_length_limit: >
+      Configure max attribute value size.
+      
+      If omitted or null, there is no limit.
+    attribute_count_limit: >
+      Configure max attribute count.
+      
+      If omitted or null, 128 is used.
   path_patterns:
     - .attribute_limits
 
 - type: Propagator
   property_descriptions:
-    composite: Configure the set of propagators to include in the composite text map propagator.
+    composite: >
+      Configure the set of propagators to include in the composite text map propagator. Values include: tracecontext, baggage, b3, b3multi, jaeger, xray, ottrace, none. For behavior of values see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration.
```

Comment:
In the spec:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#propagators-distribution

`xray` is not listed in "The official list of propagators"

Maybe clarify that propagators can be:

* from the list of official propagators (builtins)
* from extension points

Also, a propagator of `none` is dubious, this is yet another way to specify null with a different syntax.
There should be no `propagator:` node in this case.